Library/Homebrew/cmd/update.sh

```diff
@@ -556,6 +556,12 @@ EOS
     [[ -d "${DIR}/.git" ]] || continue
     cd "${DIR}" || continue
 
+    # Git's fsmonitor daemon will not release our lock unless we stop it.
+    if git fsmonitor--daemon status &>/dev/null
+    then
+      git fsmonitor--daemon stop 2>/dev/null
```

Comment:
Will it stop the daemon globally or only for the repo?

Alternatively, we can set `core.fsmonitor = false` in `.git/config` (I suppose this should exclude repo from being monitored).

Reply:
> Will it stop the daemon globally or only for the repo?

Only for the repo, so we need to do it after `cd`ing to it.

> Alternatively, we can set `core.fsmonitor = false` in `.git/config` (I suppose this should exclude the repo from monitored ones).

Yes, but there seem to be some benefits from running the daemon, so it would be nice to keep it going if possible.

Reply:
Given we're hiding the output anyway: wondering if we should just unconditionally run `git fsmonitor--daemon stop` as it's not any slower than running `git fsmonitor--daemon status`?

Reply:
Yes, running `stop` unconditionally is indeed much better.